server/docker-compose.yml

```diff
@@ -15,27 +15,19 @@ services:
       - db
     environment:
       PYTHONUNBUFFERED: 1
-      MONGO_URL: mongodb://root:toor@db
-      MONGO_DATABASE: ttt
+      DB_URI: postgresql://${APP_DB_USERNAME}:${APP_DB_PASSWORD}@db/${APP_DB_DATABASE}
+      DB_HOST: db
```

Comment:
DB_HOST?
U already have DB_URI that preset all data that needs for connection for db